ruby/.stickler.yml

```diff
@@ -2,19 +2,14 @@
 linters:
   rubocop:
     display_cop_names: true
-    # indicate where is the config file for stylelint
-    config: './rubocop.yml'
+    config: "./rubocop.yml"
 
-# add the files here you want to be ignored by stylelint
 files:
   ignore:
     - "bin/*"
     - "db/*"
     - "config/*"
     - "Guardfile"
     - "Rakefile"
+    - "node_modules/**/*"
 
-# PLEASE DO NOT enable auto fixing options
```

Comment:
@kelvin8773 well done but something went wrong during merging last changes from master :)
Can you make sure that extra comments added before are not going to disappear with your change? 🙏

Reply:
Sorry for that, I guess it is because my local repo is outdate, now should be fine now, please check again!

Kelvin

Reply:
@kelvin8773 you are still removing these lines :)

To make it more clear, I mean this text:

```
# PLEASE DO NOT enable auto fixing options
# if you need extra support from you linter - do it in your local env as described in README for this config
# find full documentation here: https://stickler-ci.com/docs
```